### ISSUE TITLE
Switching between layout recordings moves the browser window from another display

### ISSUE THREAD

**Issue by jhhom** [rxhanson/RectanglePro-Community#99]
<img width="396" alt="image" src="https://user-images.githubusercontent.com/24519631/188314606-fdaf1bdd-e44c-424f-a3fa-224d26abe806.png">

I have two saved layouts on two different displays.

I intend to use both layouts together. However, after I trigger the first layout, when I trigger the second layout, the Firefox window opened on the first display will jump to the second display.

This causes the minor inconvenience of having to drag the Firefox window back to the first display.

It's a minor inconvenience but just would like to know if there is any way to solve this?

Thanks for the awesome app.

**Comment by rxhanson** [comment]
Thanks for reporting. There's no way to distinguish windows of the same app from Rectangle Pro's perspective, other than z-index and size/location. With that in mind, it's kind of difficult to have an implementation that is simple and clearly describes which windows to put where when there are multiple windows open for an app. Right now, it will operate only on the frontmost window for an app.

Maybe it would make sense to add a source display or source z-index field to the config for each specification?

**Comment by jhhom** [comment]
Yeah I think it would make sense to add a source display to the config.
